### PR TITLE
1.1 release docs

### PR DESCRIPTION
**Pull request by bmcfee**

### CODE REVIEW

**Comment by bmcfee** [comment]
Hey folks: I'm doing some surgery to the docs / readme to make it a bit more accessible.  Let me know what you think.

**Comment by bmcfee** [comment]
Finally got the TOC organized in a sane way.  I think this is good to go, but please look it over.

File: README.md [hunk 1]
```diff
@@ -7,9 +7,33 @@ pescador
...
+
+  Pescador can dynamically interleave these sources to provide a randomized stream `D <- (A, B, C)`.
+  The distribution over `(A, B, C)` need not be uniform: you can specify any distribution you like!
+
+- Now, say you have 3000 data sources, each of which may contain a large number of samples.  Maybe that's too much data to fit in RAM at once.
+
+  Pescador makes it easy to interleave these sources while maintaining a small `working set`.
+  Not all sources are simultaneously active, but Pescador manages the working set so you don't have to.
+
+- If loading data incurs substantial latency (e.g., due to accessing storage access
```
Comment (cjacoby) [inline README.md]: End paren?

Comment (bmcfee) [reply README.md]: Copypasta error.  Fixed.

**Pull request merged**